pages/uitgifte-email.md

```diff
@@ -24,14 +24,20 @@ Wanneer u op die link klikt ontvangt u het e-mail attribuut.
 
 (Het blijkt dat in sommige e-mail lezers op mobiele telefoons deze
 link niet goed "klikbaar" weergegeven wordt. U kunt dan proberen de
-link zelf te copieëren in een webbrowser. U kunt ook proberen de mail
+link zelf te kopieëren in een webbrowser. U kunt ook proberen de mail
 op een ander apparaat, zoals een laptop of PC, te lezen. Er wordt naar
 gekeken hoe dit probleem het beste opgelost kan worden.)
 
 De geldigheid van dit e-mail attribuut is *een jaar*.
 
-De stichting Privacy by Design verwijdert dit e-mail adres na uitgifte
-uit de eigen systemen. De stichting houdt geen log bij van de
-uitgifte.
+**Note:**
+Vanaf 30 November 2020 worden e-mail credentials niet meer uitgegeven
```

Comment:
Ik zou misschien zeggen `het e-mailkaartje` of is dat niet consistent met de rest van de website?

Reply:
De rest van de pagina noemt het attribuut, wat al helemaal niet correct is imo..

Reply:
Toch maak ik er attributen van, bij nader inzien is het zo slecht nog niet